catalogbuilder/intakebuilder/getinfo.py

```diff
@@ -108,6 +108,13 @@ def getInfoFromGFDLFilename(filename,dictInfo,logger,configyaml):
             output_file_template = builderconfig.output_file_template
         except:
             sys.exit("No output_path_template found. Check configuration.")
+    if( "static" in filename ):
```

Comment:
How about limiting ourselves to `.static.` instead of `static`?  There could possibly be files matching "static" that are not static pp files (but I can't think of any..)

Reply:
```suggestion
    if( ".static" in filename ):
```